lang/en/quizaccess_seb.php

```diff
@@ -95,6 +95,30 @@
 
 // Capabilities.
 $string['seb:bypassseb'] = 'Bypass the requirement to view quiz in Safe Exam Browser.';
+$string['managetemplates'] = 'Manage Safe Exam Browser configuration templates.';
```

Comment:
Easy one, it's missing the string prefix "seb:"
```suggestion
$string['seb:managetemplates'] = 'Manage Safe Exam Browser configuration templates.';
```